readme.md

```diff
@@ -25,7 +26,7 @@ Your Answer:
 Describe at a high level how we use jQuery to submit a form via AJAX.
 
 Your Answer:
-```text
+```We would use Promise Callbacks to submit a form via AJAX
```

Comment:
What are these promise callbacks? Can you give an example? Again not looking for anything too specific, but you should be able to talk about the steps. For example: 

When submitting a form via AJAX, there are a few things we need to do:
1. Register an event listener for the form submissions (buttton click).
2. When the `submit` happens, we need to preventDefault() to keep the browser from submitting the form.
3. Next, we need to collect all the data from the form, and package it up into an object to be submitted by ajax. We can use `.val()` to get the value of an input
4. Finally, we use $.ajax() with a post/put verb, to make the ajax request. We also need to include a `data` property which contains the form data.